tox.ini

```diff
@@ -0,0 +1,27 @@
+[tox]
+envlist = py{27,34,35}-django{18,19}
+
+[testenv]
+deps =
+    -rdocs/requirements.txt
+    jinja2
+    https://github.com/tBaxter/vobject/tarball/master#egg=vobject
```

Comment:
I think the Jinja2 is related to typogrify. Without it, there are a couple of these:
```pytb
======================================================================
ERROR: test_index (happenings.tests.TestHappeningsGeneralViews)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/jwilhelm/Documents/workspace/tango-happenings/happenings/tests.py", line 30, in test_index
    resp = self.client.get(reverse('events_index'))
  File "/Users/jwilhelm/Documents/workspace/tango-happenings/.tox/py27-django19/lib/python2.7/site-packages/django/test/client.py", line 503, in get
    **extra)
  File "/Users/jwilhelm/Documents/workspace/tango-happenings/.tox/py27-django19/lib/python2.7/site-packages/django/test/client.py", line 304, in get
    return self.generic('GET', path, secure=secure, **r)
  File "/Users/jwilhelm/Documents/workspace/tango-happenings/.tox/py27-django19/lib/python2.7/site-packages/django/test/client.py", line 380, in generic
    return self.request(**r)
  File "/Users/jwilhelm/Documents/workspace/tango-happenings/.tox/py27-django19/lib/python2.7/site-packages/django/test/client.py", line 449, in request
    response = self.handler(environ)
  File "/Users/jwilhelm/Documents/workspace/tango-happenings/.tox/py27-django19/lib/python2.7/site-packages/django/test/client.py", line 123, in __call__
    response = self.get_response(request)
  File "/Users/jwilhelm/Documents/workspace/tango-happenings/.tox/py27-django19/lib/python2.7/site-packages/django/core/handlers/base.py", line 230, in get_response
    response = self.handle_uncaught_exception(request, resolver, sys.exc_info())
  File "/Users/jwilhelm/Documents/workspace/tango-happenings/.tox/py27-django19/lib/python2.7/site-packages/django/core/handlers/base.py", line 296, in handle_uncaught_exception
    return callback(request, **param_dict)
  File "/Users/jwilhelm/Documents/workspace/tango-happenings/.tox/py27-django19/lib/python2.7/site-packages/django/utils/decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "/Users/jwilhelm/Documents/workspace/tango-happenings/.tox/py27-django19/lib/python2.7/site-packages/django/views/defaults.py", line 60, in server_error
    template = loader.get_template(template_name)
  File "/Users/jwilhelm/Documents/workspace/tango-happenings/.tox/py27-django19/lib/python2.7/site-packages/django/template/loader.py", line 26, in get_template
    engines = _engine_list(using)
  File "/Users/jwilhelm/Documents/workspace/tango-happenings/.tox/py27-django19/lib/python2.7/site-packages/django/template/loader.py", line 143, in _engine_list
    return engines.all() if using is None else [engines[using]]
  File "/Users/jwilhelm/Documents/workspace/tango-happenings/.tox/py27-django19/lib/python2.7/site-packages/django/template/utils.py", line 110, in all
    return [self[alias] for alias in self]
  File "/Users/jwilhelm/Documents/workspace/tango-happenings/.tox/py27-django19/lib/python2.7/site-packages/django/template/utils.py", line 101, in __getitem__
    engine = engine_cls(params)
  File "/Users/jwilhelm/Documents/workspace/tango-happenings/.tox/py27-django19/lib/python2.7/site-packages/django/template/backends/django.py", line 31, in __init__
    options['libraries'] = self.get_templatetag_libraries(libraries)
  File "/Users/jwilhelm/Documents/workspace/tango-happenings/.tox/py27-django19/lib/python2.7/site-packages/django/template/backends/django.py", line 49, in get_templatetag_libraries
    libraries = get_installed_libraries()
  File "/Users/jwilhelm/Documents/workspace/tango-happenings/.tox/py27-django19/lib/python2.7/site-packages/django/template/backends/django.py", line 131, in get_installed_libraries
    for name in get_package_libraries(pkg):
  File "/Users/jwilhelm/Documents/workspace/tango-happenings/.tox/py27-django19/lib/python2.7/site-packages/django/template/backends/django.py", line 148, in get_package_libraries
    "trying to load '%s': %s" % (entry[1], e)
InvalidTemplateLibrary: Invalid template library specified. ImportError raised when trying to load 'typogrify.templatetags.jinja_filters': No module named jinja2
```

I'll see what I can do with python-card-me.